NOTICE

```diff
@@ -1,31 +0,0 @@
-Apache KIE
-Copyright 2024 The Apache Software Foundation
-
-This product includes software developed at
-The Apache Software Foundation (http://www.apache.org/).
-
-The Initial Developer of some parts of the framework, which are copied from, derived from, or
-inspired by KIE (Knowledge Is Everthing) group, is Red Hat, Inc (https://www.redhat.com/).
-Copyright Red Hat, Inc. and/or its affiliates.. All Rights Reserved.
```

Comment:
@jomarko Ahh, I'm sorry that I confused you. I wanted to say we can remove the items for third party components. But we need to keep the main notice about Apache KIE. So the result would be like the same as https://github.com/apache/incubator-kie-drools/blob/main/NOTICE , not removing the whole file.

Reply:
@tkobayas sorry, my fault, I added the NOTICE file back